contracts/VestingEscrowFactory.vy

```diff
@@ -1,79 +1,182 @@
-# @version 0.2.16
+# @version 0.3.7
+
 """
 @title Vesting Escrow Factory
-@author Curve Finance, Yearn Finance
+@author Curve Finance, Yearn Finance, Lido Finance
 @license MIT
-@notice Stores and distributes ERC20 tokens by deploying `VestingEscrowSimple` contracts
+@notice Stores and distributes ERC20 tokens by deploying `VestingEscrow` or `VestingEscrowFullyRevokable` contracts
 """
 
 from vyper.interfaces import ERC20
 
 
-interface VestingEscrowSimple:
+interface IVestingEscrow:
     def initialize(
-        admin: address,
         token: address,
         recipient: address,
-        amount: uint256,
         start_time: uint256,
         end_time: uint256,
         cliff_length: uint256,
+        voting_adapter_addr: address,
+    ) -> bool: nonpayable
+
+    def activate(
+        amount: uint256,
+        owner: address,
+        manager: address,
     ) -> bool: nonpayable
 
+    def get_token() -> address: view
+
+
+struct EscrowAmount:
+        escrow: address
+        amount: uint256
```

Comment:
`FundingRequest` or something similar. And looks like overindented.

Reply:
indentation is by mamushi

Reply:
Bug?)

Reply:
https://github.com/benber86/mamushi/blob/16cc0286fa7b43a9d6d7be99ac98ebed4adfd719/tests/data/declarations/struct_decl_blank_lines.vy

Reply:
Yew, quite strange thing

Reply:
struct was removed